backend/server.py

```
@@ -6493,6 +6493,151 @@ def _mask(val: str | None) -> str:
+                "error": "OpenClaw Gateway is not running",
+                "hint": (
+                    "The OpenClaw CLI may not be installed (not yet published to npm). "
+                    "The gateway subprocess starts automatically when the CLI is available. "
+                    "Meanwhile, use the Telegram bot for phone control, or set OPENCLAW_ENABLED=true "
+                    "and redeploy once the CLI is published."
+                ),
+                "status_endpoint": "/api/openclaw/status",
+                "qr_endpoint": "/api/openclaw/qr",
+            },
+        )
+    except Exception as exc:
```

Ruff (0.15.20)

[warning] Do not catch blind exception: `Exception`

(BLE001)